api/ServiceWorkerRegistration.json

```diff
@@ -753,20 +753,389 @@
             "standard_track": true,
             "deprecated": false
           }
+        },
+        "actions": {
+          "__compat": {
+            "mdn_url": "https://developer.mozilla.org/docs/Web/API/ServiceWorkerRegistration/showNotification",
+            "support": {
+              "webview_android": {
+                "version_added": "47"
+              },
+              "chrome": {
+                "version_added": "47"
+              },
+              "chrome_android": {
+                "version_added": "47"
+              },
+              "edge": {
+                "version_added": null
+              },
+              "edge_mobile": {
+                "version_added": null
+              },
+              "firefox": {
+                "version_added": false
+              },
+              "firefox_android": {
+                "version_added": false
+              },
+              "ie": {
+                "version_added": false
+              },
+              "opera": {
+                "version_added": null
+              },
+              "opera_android": {
+                "version_added": null
+              },
+              "safari": {
+                "version_added": false
+              },
+              "safari_ios": {
+                "version_added": false
+              },
+              "samsunginternet_android": {
+                "version_added": null
+              }
+            },
+            "status": {
+              "experimental": true,
+              "standard_track": true,
+              "deprecated": false
+            }
+          }
+        },
+        "badge": {
+          "__compat": {
+            "mdn_url": "https://developer.mozilla.org/docs/Web/API/ServiceWorkerRegistration/showNotification",
+            "support": {
+              "webview_android": {
+                "version_added": "53"
+              },
+              "chrome": {
+                "version_added": "53"
+              },
+              "chrome_android": {
+                "version_added": "53"
+              },
+              "edge": {
+                "version_added": null
+              },
+              "edge_mobile": {
+                "version_added": null
+              },
+              "firefox": {
+                "version_added": false
+              },
+              "firefox_android": {
+                "version_added": false
+              },
+              "ie": {
+                "version_added": false
+              },
+              "opera": {
+                "version_added": "39"
+              },
+              "opera_android": {
+                "version_added": "39"
```

Comment:
Opera happens to be correct in this case. Generally speaking, if the Opera version number is not 13 behind the Chrome version number, one of them is probably wrong. If you can't can't get an immediate answer on [Chrome Status](https://www.chromestatus.com/features), ping me and I'll research it.